test/integration/query-interface/getForeignKeyReferencesForTable.test.js

```diff
@@ -40,5 +40,51 @@ describe(Support.getTestDialectTeaser('QueryInterface'), () => {
       expect(refs[0]).deep.include.all(expectedObject);
 
     });
+
+    describe('with schemas', () => {
+      // MariaDB does not really support schemas (they are synonyms of databases)
```

Comment:
Is this behaviour different between mariadb and mysql?

Reply:
For MySQL, `Support.sequelize.dialect.supports.schemas` is `false`, but for MariaDB it's true (even though it does not really support schemas).

You can see it there:
1. [For MySQL](https://github.com/sequelize/sequelize/blob/v6/src/dialects/mysql/index.js)
2. [For MariaDB](https://github.com/sequelize/sequelize/blob/v6/src/dialects/mariadb/index.js#L39)